hackathon_site/hackathon_site/urls.py

```diff
@@ -15,8 +15,41 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf.urls import url
+
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Snippets API",
+        default_version="v1",
+        description="Test description",
+        terms_of_service="https://www.google.com/policies/terms/",
+        contact=openapi.Contact(email="contact@snippets.local"),
+        license=openapi.License(name="BSD License"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+    #    url="/api/",
```

Comment:
I see that this is being weird, and that we would need the full URL path. Let's see if the base path fixes itself magically once some other endpoints are included.